content/docs/command-reference/import.md

```diff
@@ -11,7 +11,7 @@ the import.
 
 ```usage
 usage: dvc import [-h] [-q | -v]
-                  [-o <path>] [--file <filename>] [--rev <commit>]
+                  [-o <path>] [--file <filename>] [--rev <commit>] [--no-exec]
                   url path
```

Comment:
That new line is too long. Let's reformat this way please:
```
usage: dvc import [-h] [-q | -v] [-o <path>] [--file <filename>]
                  [--rev <commit>] [--no-exec]
                  url path
```

Reply:
I've moved it to a new line.